### PR TITLE
docs: Add code example for modelling organizations guide

### PR DESCRIPTION
**Pull request by sixhobbits**
Depends on https://github.com/ritza-co/example-modeling-organizations being moved/forked to the Fusionauth GitHub org.

### CODE REVIEW

**Comment by mark-robustelli** [comment]
@sixhobbits , I have moved the application to the FusionAuth org - https://github.com/FusionAuth/fusionauth-example-modeling-organizations.  I received the following errors:
```
express-session deprecated undefined resave option; provide resave option app.js:41:9
express-session deprecated undefined saveUninitialized option; provide saveUninitialized option app.js:41:9
```
but when I ran npm audit fix --force it appeared to work.  Please update.

File: astro/src/content/docs/extend/examples/modeling-organizations.mdx [hunk 1]
```diff
@@ -214,23 +244,39 @@ Here's an example of the response from the [Retrieve Grants](/docs/apis/entities
 }
 ```
 
-This response indicates that the User belongs to 3 organizations and has various permissions for each. To manage this in your AwesomeCRM application, you can present an option to the user when they log in, to select the company they'd like to work on initially. This option screen might look like this:
+This response indicates that the user belongs to three organizations and has different permissions for each. To manage this in your AwesomeCRM application, you can provide users with an option to select the company they would like to work on when they log in, as in the example below.
```
Comment (mark-robustelli) [inline astro/src/content/docs/extend/examples/modeling-organizations.mdx:247]: In the example app, richard is the user with multiple companies.
Screenshot 2024-09-12 at 4 42 39 PM

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Overall this looks pretty good, just needs a few minor modifications.

File: astro/src/content/docs/extend/examples/modeling-organizations.mdx (hunk 1 above)
Comment (rideam) [reply astro/src/content/docs/extend/examples/modeling-organizations.mdx:247]: I have updated the image

**Comment by rideam** [comment]
@mark-robustelli I opened a PR here https://github.com/FusionAuth/fusionauth-example-modeling-organizations/pull/2 to fix the warnings 

```
express-session deprecated undefined resave option; provide resave option app.js:41:9
express-session deprecated undefined saveUninitialized option; provide saveUninitialized option app.js:41:9
```

**Review by mark-robustelli — APPROVED** [review]

**Pull request merged**